### DISCUSSION TITLE
i search for a file but the site is not there anymore

### DISCUSSION THREAD

**Discussion by Ensar383** [community/community#126864] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

hello,
can u guys help me pls i search for a file to jailbreak my switch but the download link doesnt exist anymore. the download should be on this link : https://github.com/ITotalJustice/sigpatch-updater/releases/tag/v0.2.0 . It would nice if u guys could help me. :)

**Comment by CopperEagle** [comment]
Hey @Ensar383 

If you need this for your Nintendo Switch, probably [this](https://github.com/HamletDuFromage/AIO-switch-updater) is what you are looking for. Just go to the "releases" tab and see what you need. No warranty from my side though.

The way I found it is from [this](https://github.com/HamletDuFromage/sigpatches-updater) which is archived since 2021. 

Is that what you are looking for?

**Reply by Ensar383** [reply]
Thanks but i still cant find it i search for the sigpatch-Updater v0.2.0

**Reply by CopperEagle** [reply]
Hey @Ensar383 

[Here](https://github.com/HamletDuFromage/sigpatches-updater/releases/tag/0.2) you go. Just note that this is an old release from 2020. The repository is not maintained anymore. Newer versions are available in the repository I linked above.

If this answers your question, consider marking it as the answer. Thanks!

**Comment by Ayoubbenderdouch** [comment]
yes
